libtransact/src/protocol/batch.rs

```diff
@@ -142,6 +142,10 @@ impl Batch {
         &self.header
     }
 
+    pub fn parse_header(&self) -> Result<BatchHeader, BatchBuildError> {
```

Comment:
Should the commit say Add `parse_header` ?

Reply:
Oops, meant to change this. Updated the method to `deserialized_header`.